ksql-parser/src/main/java/io/confluent/ksql/parser/tree/TableElement.java

```diff
@@ -38,6 +39,7 @@
   private final Namespace namespace;
   private final ColumnName name;
   private final Type type;
+  private final Optional<SourceName> source;
```

Comment:
If we're adding a new field it should be part of `equals`, `hashCode` and `toString`.